### PR TITLE
ENTSWM-136: Productised booster

### PR DESCRIPTION
**Pull request by heiko-braun**
HOLD: This is for reviewing purposes only

### CODE REVIEW

File: .openshiftio/booster.yaml [hunk 1]
```diff
@@ -1,2 +1,7 @@
 name: WildFly Swarm - HTTP
 description: Runs a WildFly Swarm HTTP application
+versions:
```
Comment (heiko-braun) [inline .openshiftio/booster.yaml:3]: @kenfinnigan @Ladicek What terms should we use here? community/redhat was mentioned on the ML, but I don't know if this has been decided.

File: pom.xml [hunk 2]
```diff
@@ -162,5 +163,41 @@
...
+            <scope>import</scope>
+          </dependency>
+        </dependencies>
+      </dependencyManagement>
+
+      <repositories>
+        <!-- PNC + brew + maven central -->
+        <repository>
+          <id>shared-imports</id>
+          <url>http://indy.cloud.pnc.engineering.redhat.com/api/group/builds-untested+shared-imports+public/</url>
```
Comment (heiko-braun) [inline pom.xml]: The URL's obviously need to change once we are on MRCC

**Comment by heiko-braun** [comment]
@kenfinnigan Let's review a single instance first and come to a conclusion before we touch all the other boosters.

File: pom.xml (hunk 2 above)
Comment (fantasianplus) [inline pom.xml]: Publishing internal URLs is a no-no.

Comment (Ladicek) [reply pom.xml]: Publishing internal URLs is a no-no.

Comment (heiko-braun) [reply pom.xml]: @ fantasianplus  So, who is fantasianplus?

Comment (Ladicek) [reply pom.xml]: Sorry, that's my scratch GitHub account for OpenShift.io purposes :-)

**Comment by heiko-braun** [comment]
@Ladicek it now points to https://maven.repository.redhat.com/ga/

File: .openshiftio/booster.yaml (hunk 1 above)
Comment (Ladicek) [reply .openshiftio/booster.yaml:3]: I think `community` and `redhat` are OK and there's a semi-agreement on those terms I think.

File: pom.xml [hunk 3]
```diff
@@ -162,5 +163,52 @@
...
+
+      <repositories>
+        <repository>
+          <releases>
+            <enabled>true</enabled>
+          </releases>
+          <snapshots>
+            <enabled>false</enabled>
+          </snapshots>
+          <id>boss-enterprise-maven-repository</id>
```
Comment (Ladicek) [inline pom.xml]: Probably should have been `jboss` :-) The plugin repository as well.

File: .openshiftio/booster.yaml (hunk 1 above)
Comment (kenfinnigan) [reply .openshiftio/booster.yaml:3]: Not sure if we need to get JohnC to ok the user facing names for the versions or not?

Also, raised a question on ML as to whether profiles can ever work as I had forgotten about the openshift profile

**Comment by heiko-braun** [comment]
@kenfinnigan the latest commit cleans everything as discussed and introduced `pom-redhat.xml` next to `pom.xml`. With the changes to support `mvn -f pom-redhat.xml` in wildfly-swarm core, everything should be in place to make this work. But currently we are in a chicken and egg situation, with respect to testing the productised boosters: The product branch needs to updated to include the support for `mvn -f` before we can verify this work in product boosters.

**Pull request closed without merging**